CHANGELOG.md

```diff
@@ -3,3 +3,4 @@
   - Add the number of hits in search result (#541)
   - Add support for aligned crop in search result (#543)
   - Sanitize the content displayed in the web interface (#539)
+  - Add support of nested null values
```

Comment:
Thank you, would it be possible to also specify that we support boolean values too? I forgot to change it on the other PR :)

```suggestion
  - Add support of nested null and boolean values
```